lib/src/base-styles/_nx-btn.scss

```diff
@@ -11,107 +11,110 @@
   @include semi-bold();
   @include container-horizontal;
 
-  background-color: $nx-grey-1;
-  border: $nx-border-dark;
-  border-radius: 4px;
-  color: $nx-text-color-light;
+  background-color: $nx-white;
+  border: 1px solid #045dd1;
+  border-radius: 6px;
+  color: #045dd1;
   cursor: pointer;
   display: inline-block;
-  font-size: 14px;
-  height: 32px;
+  font-size: 16px;
+  height: 40px;
```

Comment:
The fact that these are 40px tall is reminiscient of my recent work on the NxAlerts.  Should we try to be consistent about our approaches between the two?  With the alerts, I avoided using an explicit height and rather relied on the default line-height of the text and some standard-variable-based top and bottom padding.

Reply:
> The fact that these are 40px tall is reminiscient of my recent work on the NxAlerts. Should we try to be consistent about our approaches between the two? With the alerts, I avoided using an explicit height and rather relied on the default line-height of the text and some standard-variable-based top and bottom padding.

I get to 42px with default values, I think that's near enough: https://github.com/sonatype/sonatype-react-shared-components/pull/39/commits/90a22b5e75730be70b37ebf362972aea477b3b7a